Makefile

```
@@ -1,9 +1,35 @@
-all: test lint
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+##### Help message #####
+help:  ## Display this help
+	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make <target> \033[36m\033[0m\n\nTargets:\n"} /^[a-zA-Z0-9_-]+:.*?##/ { printf "  \033[36m%-20s\033[0m %s\n", $$1, $$2 }' $(MAKEFILE_LIST)
+
+BUILD_FLAGS =
+COMPOSE ?= docker-compose
+
+all: build up test lint down
```

No need to spin up the docker container if the linter fails. 
```suggestion
all: build lint up test down
```